ci-operator/step-registry/rosa/cluster/wait-ready/cluster/rosa-cluster-wait-ready-cluster-commands.sh

```
@@ -244,7 +258,7 @@ if [[ "$FAILED_INSTALL" == "yes" ]]; then
   status_desc=$(jq -r '.status.description // "N/A"' "${ARTIFACT_DIR}/cluster-description.json" 2>/dev/null || echo "N/A")
   log "Cluster status description: ${status_desc}"
   # Save install logs
-  rosa logs install -c ${CLUSTER_ID} > "${ARTIFACT_DIR}/.install.log" || true
+  timeout 60 rosa logs install -c ${CLUSTER_ID} > "${ARTIFACT_DIR}/.install.log" || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)